src/main/scala/com/gu/invoicing/invoice/Model.scala

```diff
@@ -83,7 +83,7 @@ object Model extends OptionPickler {
     date: LocalDate,
     paymentMethod: PaymentMethod,
     price: Double,
-    pdfFileId: String, // used to download the file
+    pdfFileUrl: String, /* invoices/{accountId}/{fileId} */
```

Comment:
this still isn't really a URL and is relative to this api (rather the user-facing `manage-frontend` host) I think exposing just the `pdfFileId` from the api is neatest, if you really want to expose a path though, perhaps...
```suggestion
    pdfFilePath: String, /* invoices/{accountId}/{fileId} */
```

Reply:
lol you'd already done it while i was typing - https://github.com/guardian/invoicing-api/pull/7/commits/01ac1422b0d378832a3894dd808c3ebcf5a69909